### ISSUE TITLE
Minor and best practice issues

### ISSUE THREAD

**Issue by michellenSD** [logicly-au/docs.iar-dst.online#41]
**28. List mark-up used incorrectly**
1-2h _(Minor, Best practice)_

**30. Note: Missing Skip to content link**
\>4h _(Major, Best practice)_

**31. Note: Inappropriate label for navigation landmarks**
2-3h _(Minor, Best practice)_

**33. Note: Content not hidden for screen reader users**
1-2h _(Minor, Best practice)_

**49. Hidden content receives keyboard focus**
1-2h _(Minor, A)_

**50. Unnecessary alt text for decorative images**
<1h _(Minor, A)_

**51. Note: Missing label for navigation region**
1-2h _(Minor, Best practice)_

See https://gh.sdintra.net/IAR-DST/iar-dst-app/issues/288 for report
